tests/test___init__.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the functions contained in pennylane/__init__.py.
+"""
+# pylint: disable=protected-access,cell-var-from-loop
+
+import pytest
+import pennylane as qml
+
+
+class TestPennyLaneInit:
+    """Unit tests for the functions in the pennylane/__init__.py."""
+
+    pytest.importorskip("pennylane_qiskit")
```

Comment:
Is this doing what we think it is? (as it's not acting as a decorator?)

Reply:
(I've removed this test for now.)

For me, it seemed to have worked fine locally